automerge-backend/src/columnar.rs

```diff
@@ -429,9 +429,11 @@ impl<'a> Iterator for ValueIterator<'a> {
             }
             v if v % 16 == VALUE_TYPE_BYTES => {
                 let len = v >> 4;
-                let _data = self.val_raw.read_bytes(len).ok()?;
-                unimplemented!()
-                //Some((amp::Value::Bytes(data))
+                let data = self.val_raw.read_bytes(len).ok()?;
+                Some(amp::ScalarValue::Bytes(
+                    // This unwrap is fine b/c we encode as valid UTF-8 when serializing
+                    String::from_utf8(data.into()).unwrap(),
```

Comment:
This is only safe if we know we encoded it but someone else could have so I think this should return the error.

Reply:
Closing b/c new version stores raw bytes (no string).